reasoner/processor/AbstractProcessor.java

```diff
@@ -182,6 +186,36 @@ public Driver<Monitor> monitor() {
             return monitor;
         }
 
+        public ReasonerPerfCounters perfCounters() {
+            return perfCounters;
+        }
+
+        public static class ReasonerPerfCounters {
+
+            public static final String KEY_TIME_PLANNING_MS = "time_planning_ms";
+            public static final String KEY_COUNT_MATERIALISATIONS = "count_materialisations";
+            public static final String KEY_COUNT_CONJUNCTION_PROCESSORS = "count_conjunction_processors";
+            public static final String KEY_COUNT_COMPOUND_STREAMS = "count_compound_streams";
+
```

Comment:
Moved perf-counters here and made the keys static strings. Could also make them enums?